docs/content/tutorial_code/deploying_with_gloo.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Deploying Filters to Gloo"
+weight: 2
+description: Deploy a wasm filter using Gloo as the control plane.
+---
+
+In this tutorial we'll deploy an existing WebAssembly (WASM) module from [the WebAssembly Hub](https://webassemblyhub.io) directly to Envoy via [Gloo](https://docs.solo.io/gloo/latest) installed to our kubernetes cluster.
+
+## Prepare environment
+
+To get started, let's deploy a sample service that we can call through Envoy. We'll deploy the sample petstore API:
+
+```shell
+kubectl apply -f \
+https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
+```
+
+You should now have the petstore running:
+
+```shell
+kubectl get po 
+```
+
+```
+
+NAME                        READY   STATUS    RESTARTS   AGE
+petstore-5dcf5d6b66-n8tjt   1/1     Running   0          2m20s
+```
+
+### Deploying Envoy
+
+In this tutorial, we'll use Gloo, an API Gateway based on Envoy that has built-in wasm support but these steps should also work for base Envoy.
+
+First, install Gloo via the helm chart:
+
+```shell
+helm repo update
+kubectl create ns gloo-system
+helm install gloo-gateway gloo/gloo --namespace gloo-system \
+  --set global.wasm.enabled=true
+```
+
+Gloo will be installed to the `gloo-system` namespace.
+
+### Verify set up
+
+Lastly, we'll set up our routing rules to be able to call our `petstore` service. Let's add a route to the routing table:
+
+Download and apply the [virtual service manifest](../default-virtualservice.yaml)
+```shell
+kubectl apply -f default-virtualservice.yaml
+```
+
+To get Gloo's external IP, run the following:
+
+```shell
+URL=$(kubectl get svc -n gloo-system gateway-proxy \
```

Comment:
super nit: GLOO_URL
don't wanna take the chance of overwriting something that's there

Reply:
i prefer URL here as it's more translatable across tutorials with Istio